xml/System/Span`1.xml

```diff
@@ -607,7 +607,7 @@ The `GetPinnableReference` method returns a [ref struct](~/docs/csharp/reference
         <value>The element at the specified index.</value>
         <remarks>To be added.</remarks>
         <exception cref="T:System.IndexOutOfRangeException">
-          <paramref name="index" /> is less than zero or greater than or equal to <see cref="P:System.Span`1.Length" />.</exception>
```

Comment:
This is an incorrect change and should be reverted. The indexer will fail for equal to:

> Thrown when index less than 0 or index greater than or equal to Length
https://github.com/Anipik/coreclr/blob/83fc9359bbe09e10235545749b7794fa6dd1f114/src/System.Private.CoreLib/shared/System/Span.Fast.cs#L139-L155